sdlf-pipLibrary/external_wheels.py

```diff
@@ -0,0 +1,62 @@
+"""
+Upload the Glue Wheel for the listed external libraries
+
+MANDATORY args:
+1 - Auxiliary S3 bucket
+
+E.g. python3 MY_BUCKET_NAME
+"""
+
+import sys
+import json
+import urllib
+from time import sleep
+from io import BytesIO
+
+import boto3
+import botocore
+
+def add_wheels(name, info, team_name, bucket, client_s3, client_ssm):
+    url = info["url"]
+    file_name = url.rsplit(sep="/", maxsplit=1)[1]
+    print(f"file_name: {file_name}")
+    key = f"common_wheels/{team_name}/{file_name}"
+    print(f"key: {key}")
+
+    try:
+        print(f"Downloading wheel: {file_name}")
+        print(f"url: {url}")
+        response = urllib.request.urlopen(url)
+        print(f"Uploading wheel: {file_name}")
+        client_s3.put_object(Body=BytesIO(response.read()),
+                             Bucket=bucket,
+                             Key=key)
+        print(f"s3://{bucket}/{key} uploaded")
+        sleep(5)
+    except botocore.exceptions.ClientError as ex:
+        raise ex
+
+
+    client_ssm.put_parameter(
+        Name=f"/SDLF/Common/{team_name}/{name}",
```

Comment:
And this: `f"/SDLF/Wheels/{team_name}/{name}"`

Reply:
updated to wheels